frontend/src/translations/ru/common.json

```
@@ -347,6 +349,9 @@
         },
         "DeleteKey": {
           "key_deleted": "Ключ удалён: ${public_key}..."
+        },
+        "delegate": {
```

Let's use PascalCase here as we used before: 
```suggestion
        "Delegate": {
```